src/common/app.config.js

```diff
@@ -78,8 +78,326 @@ const appConfig = /* @ngInject */ function (envServiceProvider, $compileProvider
+    EMAIL_LENGTH_ERROR: 'Este campo no puede tener más de 100 caracteres.',
+    BANK_NAME_ERROR: 'Debes ingresar un nombre bancario',
+    MAX_LENGTH_BANK_NAME_ERROR: 'El nombre del banco no puede tener más de 30 caracteres.',
+    ACCOUNT_TYPE_ERROR: 'Debes elegir un tipo de cuenta',
+    ROUTING_NUM_ERROR: 'Debe ingresar un número de ruta',
+    MIN_LENGTH_ROUTING_ERROR: 'Este número de ruta debe contener 9 dígitos.',
+    ROUTING_INVALID_ERROR: 'Este número de ruta no es válido.',
+    ACCOUNT_NUM_ERROR: 'Debes ingresar un número de cuenta',
+    MIN_LENGTH_ACCOUNT_ERROR: 'Este número de cuenta debe contener 2 o más dígitos.',
+    MAX_LENGTH_ACCOUNT_ERROR: 'Este número de cuenta debe contener 17 dígitos o menos',
+    RETYPE_ACCT_NUM: 'Debe volver a escribir su número de cuenta',
+    ACCT_NUM_MISMATCH: 'Sus números de cuenta no coinciden',
+    WHERE_TO_FIND_BANK_NUMS_TITLE: 'Dónde encontrarlo',
+    WHERE_TO_FIND_BANK_NUMS_MESSAGE: 'Use su cuenta y números de ruta de su cheque. No use su comprobante de depósito.',
+    MUST_AGREE_BANK_ACCT: 'Debe aceptar este acuerdo para pagar con una cuenta bancaria',
+    CARD_NUM_ERROR: 'Debes ingresar un número de tarjeta',
+    MIN_LENGTH_CARD_NUM_ERROR: 'Este número de tarjeta debe contener al menos 13 dígitos',
+    MAX_LENGTH_CARD_NUM_ERROR: 'Este número de tarjeta no puede contener más de 16 dígitos.',
+    CARD_TYPE_ERROR: 'Este tipo de tarjeta no se reconoce',
+    INVALID_TYPE_ERROR: 'Este es un número no válido {{cardType}}. Debe tener {{cardDigits}} dígitos.',
+    INVALID_CARD_NUM_ERROR: 'Este número de tarjeta no es válido. Al menos un dígito se ingresó incorrectamente.',
+    CARD_NAME_ERROR: 'Debes ingresar el nombre en la tarjeta',
+    MAX_LENGTH_CARD_NAME_ERROR: 'Este nombre no puede tener más de 50 caracteres.',
+    MONTHS: {
+      JAN: 'Enero',
+      FEB: 'Febrero',
+      MAR: 'Marzo',
+      APR: 'Abril',
+      MAY: 'Mayo',
+      JUN: 'Junio',
+      JUL: 'Julio',
+      AUG: 'Agosto',
+      SEP: 'Septiembre',
+      OCT: 'Octubre',
+      NOV: 'Noviembre',
+      DEC: 'Diciembre'
+    },
+    CARD_EXP_MONTH_ERROR: 'Debes elegir un mes de vencimiento',
+    CARD_EXPIRED_ERROR: 'Su tarjeta de crédito ha caducado o seleccionó el mes o año incorrecto',
+    CARD_EXP_YEAR_ERROR: 'Debes elegir un año de vencimiento',
+    CARD_SEC_CODE_ERROR: 'Debes ingresar el código de seguridad',
+    MIN_LENGTH_CARD_SEC_CODE: 'El código de seguridad debe tener al menos 3 dígitos.',
+    MAX_LENGTH_CARD_SEC_CODE: 'El código de seguridad no puede tener más de 4 dígitos.',
+    LOCATION_OF_CODE_AMEX: 'Código de 4 dígitos en el frente de la tarjeta',
+    LOCATION_OF_CODE_OTHER: 'Código de 3 dígitos en el reverso de la tarjeta',
+    LOADING_ADDRESS: 'Cargando dirección postal...',
+    COUNTRY_LIST_ERROR: 'There was an error loading the list of countries. If you continue to see this message, contact <a href="mailto:eGift@cru.org">eGift@cru.org</a> for assistance.',
```

Comment:
@moeloubani this one isn't Spanish. Did this and the ones below get missed? Or intentionally deferred?

Reply:
Some of the translations have come yet from the global team working on them but we wanted to get the form out with the incomplete translations for now and add those in once we get them

Reply:
great